spec/03-plugins/38-ai-proxy/02-openai_integration_spec.lua

```diff
@@ -1444,7 +1444,7 @@ for _, strategy in helpers.all_strategies() do if strategy ~= "cassandra" then
         first_got.usage.time_per_token = 1
 
         assert.same(first_expected, first_got)
-        assert.is_true(actual_llm_latency > 0)
+        assert.is_true(actual_llm_latency >= 0)
```

Comment:
Is these any possible equal to 0?

Reply:
Yes. This latency is calculated with `ngx.now()` and not any monotonic time source, so it's always possible for it to be 0, especially in tests where things are mocked/no actual proxying happens.

https://github.com/Kong/kong/blob/846f22288814d37956a295c6d5bb846d63dbb180/kong/llm/drivers/shared.lua#L660

https://github.com/Kong/kong/blob/846f22288814d37956a295c6d5bb846d63dbb180/kong/llm/drivers/shared.lua#L718